Core/include/Acts/TrackFitting/Chi2Fitter.hpp

```diff
@@ -706,6 +722,7 @@ class Chi2Fitter {
                              c2rCurrent.covariance.inverse() *
                              c2rCurrent.residuals;
       ACTS_VERBOSE("chi2 | it=" << i << " | χ² = " << c2rCurrent.chisquare);
+      // chisquare is here the global chisquare
```

Comment:
maybe move this comment up or down a few lines to make clear where it belongs to